src/docs/development/tools/devtools/memory.md

```diff
@@ -5,41 +5,68 @@ description: Learn how to use the DevTools memory view.
 
 ## What is it?
 
-Allocated Dart objects created using a class constructor
-(for example, by using `new MyClass()` or `MyClass()`)
-live in a portion of memory called the heap.
+Allocated Dart objects created using a class constructor (for
+example, by using `new MyClass()` or `MyClass()`) live in a partion
+of memory called the heap that is managed by the Dart VM (virtual
+machine). 
 
 ## DevTools memory page
 
 DevTools Memory page lets you peek at how an isolate is using
-memory at a given moment. Using Monitor Allocations and
-Reset options, you can analyze accumulator counts. The accumulators
-can be used to study the rate of memory allocations,
-if you suspect your application is leaking memory or has
-other bugs relating to memory allocation.
-
-A Snapshot is a complete view of all objects in the Dart memory heap.
-Each time a snapshot is taken, an analysis is performed over the memory
-data.  The analysis attempts to identify any memory patterns that may
-cause leaks or lead to application crashes. For example, loading large
-assets for thumbnail-sized images in inefficient, memory usage can
-be improved by loading smaller assets or adjusting the cacheWidth/cacheHeight
-to decode an image to a smaller size reducing the memory usage of the
-ImageCache. The analysis catches issues like this.
-
-Memory profiling consists of six parts, each increasing
-in granularity:
+memory at a given moment. 
+
+Memory profiling consists of six parts, each increasing in granularity:
```

Comment:
I think we should condense this and the bullets below. 
```
Memory profiling in DevTools consists of 3 main functions:
* Monitor the memory overview chart for usage statistics and events
* View memory snapshots and analyses to detect possible memory issues and inspect objects
* Monitor and track allocations for specific classes and objects

## Memory Overview Chart
<insert all the information about the overview chart, including the memory usage statistics, events, etc.>

## Snapshot and Analysis
< insert all info about snapshot, analysis, etc. including any functions of the controls that are associated with this data >

## Allocation Monitoring and Tracking
< insert all info about monitoring and tracking allocations >
```

Reply:
I've reorganized the doc all the information was added but was a bit scattered. Better flow.